zoleotracker/databaseSQL.py

```diff
@@ -1,57 +1,62 @@
-import mysql.connector
-from mysql.connector import Error
-
-def create_db_connection(host_name, user_name, user_password, db_name):
-    connection = None
-    try:
-        connection = mysql.connector.connect(
-            host=host_name,
-            user=user_name,
-            passwd=user_password,
-            database=db_name
-        )
-        print("MySQL Database connection successful")
-    except Error as err:
-        print(f"Error: '{err}'")
+import sqlite3
+import pandas as pd
 
+def create_db_connection():
+    connection = sqlite3.connect("zoleo.db")
     return connection
 
-def check_for_table(connection, table_name):
-    cursor = connection.cursor()
-    try:
-        cursor.execute("SHOW TABLES LIKE '%s' ", table_name)
-        result = cursor.fetchall()
-        if table_name in result:
-            return True
-        return False
-    except Error as err:
-        print(f"Error: '{err}'")
-
-def execute_query(connection, query):
-    cursor = connection.cursor()
-    try:
-        cursor.execute(query)
-        connection.commit()
-        print("Query successful")
-    except Error as err:
-        print(f"Error: '{err}'")
-
-def read_query(connection, query):
-    cursor = connection.cursor()
-    result = None
-    try:
-        cursor.execute(query)
-        result = cursor.fetchall()
-        return result
-    except Error as err:
-        print(f"Error: '{err}'")
-
-def execute_list_query(connection, sql_query, query_values):
-    cursor = connection.cursor()
-    try:
-        cursor.executemany(sql_query, query_values)
-        connection.commit()
-        print("Query successful")
-    except Error as err:
-        print(f"Error: '{err}'")
+def create_table(connection):
```

Comment:
create if not exists is a common function name
```suggestion
def create_table_if_not_exists(connection):
```